internal/layers/layers.go

```diff
@@ -123,10 +123,22 @@ func MountLCOWLayers(ctx context.Context, containerID string, layers *LCOWLayers
 	if err := wclayer.GrantVmAccess(ctx, vm.ID(), hostPath); err != nil {
 		return "", "", nil, err
 	}
+	mConfig := &scsi.MountConfig{
+		Encrypted: vm.ScratchEncryptionEnabled(),
+		Verity:    scsi.ReadVerityInfo(ctx, hostPath),
+		// For scratch disks, we want to make sure that the disk is formatted
+		// with a filesystem.
```

Comment:
This comment is kind of confusing, because it implies for non-scratch disks, do we _not_ care if they have a filesystem? No, rather that for scratch we support formatting it if it's unformatted, whereas other disks we assume will already have a filesystem present.

Reply:
Updated.